app/controllers/accounts_controller.rb

```diff
@@ -17,7 +17,7 @@ def show
   end
 
   def create
-    @account = Account.new(account_params.merge(family: current_family))
+    @account = Account.new(account_params.merge(family: Current.family))
```

Comment:
In the Current [docs](https://api.rubyonrails.org/classes/ActiveSupport/CurrentAttributes.html) they show how you can make this even cleaner by setting the default on the model using `Current` instead of having to merge it in with params. 

```ruby
belongs_to :creator, default: -> { Current.user }
```

Reply:
I was about to follow up with another PR that adds test for this controller and then uses the association fully...

```rb
@account = Current.family.accounts.new(account_params)
```

I think that's the clearest option. I personally quite like the `default:` option though.